### ISSUE TITLE
Telegram icon doesn't match the look of OS X

### ISSUE THREAD

**Issue by inlinecoder** [overtake/telegram#267]
Following the guidelines is saint thing, so it will be better if our favourite messenger would have a nice, native look icon in our docks that persists there for the whole day/month/year.

BEFORE:
<img width="1492" alt="screen shot 2016-05-17 at 1 43 04 pm" src="https://cloud.githubusercontent.com/assets/4806743/15319834/97679be0-1c35-11e6-86ea-9fd1dfaad144.png">

AFTER:
<img width="1464" alt="screen shot 2016-05-17 at 1 43 41 pm" src="https://cloud.githubusercontent.com/assets/4806743/15319839/9b64d492-1c35-11e6-977b-ee3ca751d581.png">

If you'll ever decide to change it, I'll provide any necessary assets for that.

**Comment by brainlessgit** [comment]
:+1:

**Comment by caiodias** [comment]
Did you download the app from Mac App Store?

**Comment by inlinecoder** [comment]
@caiodias , yes, Telegram is downloaded from AppStore, but I replaced the default icon manually.

**Comment by grigala** [comment]
I'm not sure about this one but Telegram definitely needs to have OSX dark theme matching.
